web/app.go

```diff
@@ -197,6 +197,11 @@ func NewAppWithDeps(config *Config, deps Dependencies) (*App, error) {
 		apiGroup.GET("/checks/catalog", ApiChecksCatalogHandler(deps.checksService))
 	}
 
+	internalApi := webEngine.Group("/api/internal")
+	{
+		internalApi.GET("/clusters/settings", ApiGetClustersSettingsHandler(deps.clustersService))
+	}
+
```

Comment:
I don't think we shouldn't have an `internal` group on the `webEngine` GIN instance: we established in the past that internal APIs would go in what is currently called the `collectorEngine`, which has mTLS, and should be renamed to something like `internalEngine` soon enough.
I would suggest to put this endpoint in the `apiGroup` like all the others, for simplicity, and if we need to review grouping then let's see the global picture and rethink all of the endpoints.

Reply:
Good to me.
shall we keep `/api/internal/clusters/settings` or do you prefer `/api/clusters/settings`?

Reply:
I agree with this one, I would defer the `internalAPI` topic a bit and add it to the standard `apiGroup` for now

Reply:
@nelsonkopliku `/api/clusters/settings` sounds good to me

Reply:
Deal!